collections/oneruntime/general-notifications/event-general-notifications/batch-status-updated/event-general-notifications.json

```diff
@@ -24,12 +24,38 @@
           },
           "engagementTemplates": [
             {
-              "path": "/batch-status-updated/in-app-notification/en_US/engagement-template.json",
+              "path": "/batch-status-updated/in-app-notification/en_US/batch-engagement-template.json",
               "templateName": "batch-status-updated-in-app-notification",
               "locale": "en_US"
             }
           ]
         },
+        {
```

Comment:
if my eyes see right, it should not be here in the middle of `deliveryOptions` but should be bottom, no?

Reply:
Because both notifications for direct debit and batches comes from same service, we have to distinguish them with separate filter logic. Having this in mind I had to implement it as separate delivery option. Did I answered your doubts?

Reply:
yes, it won't work. Please check this [example](https://github.com/Backbase/engagements-data/blob/master/collections/oneruntime/general-notifications/event-general-notifications/audit-export-completed/event-general-notifications.json) it may help you

Reply:
I see now, I believe it could create a lot of problems. Thank you a lot for spotting this!